ztp/policygenerator-kustomize-plugin/testPolicyGenTemplate/group-du-sno-validator-ranGen.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  # The name will be used to generate the placementBinding name as {name}-placementBinding, the placementRule name as {name}-placementRule,
+  # and the policy name as {name}-{spec.sourceFiles[x].policyName}
+  name: "group-du-sno-validator"
+  namespace: "ztp-group"
```

Comment:
Can you make this look more like test code? When the test versions are nearly identical to the reference configuration it causes confusion on what the active version of code is.